src/CFBG_SC.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (С) since 2019 Andrei Guluaev (Winfidonarleyan/Kargatum) https://github.com/Winfidonarleyan 
+ *  Copyright (С) since 2019 Andrei Guluaev (Winfidonarleyan/Kargatum) https://github.com/Winfidonarleyan
```

Comment:
Let's also add the AC copyright as more people are working on it, but keep the original author who is @Winfidonarleyan